django/core/management/__init__.py

```diff
@@ -118,17 +119,29 @@ def call_command(command_name, *args, **options):
     }
     arg_options = {opt_mapping.get(key, key): value for key, value in options.items()}
     parse_args = [str(a) for a in args]
+
+    # Return parser actions and sub parser choices actions.
+    def get_actions(parser):
+        actions = []
+        for opt in parser._actions:
+            actions.append(opt)
+            if isinstance(opt, _SubParsersAction):
+                for _, sub_opt in opt.choices.items():
+                    actions += get_actions(sub_opt)
+        return actions
+
+    parser_actions = get_actions(parser)
     # Any required arguments which are passed in via **options must be passed
     # to parse_args().
     parse_args += [
         '{}={}'.format(min(opt.option_strings), arg_options[opt.dest])
-        for opt in parser._actions if opt.required and opt.dest in options
+        for opt in parser_actions if opt.required and opt.dest in options if opt.option_strings
```

Comment:
New tests pass without this change.

Reply:
I changed the new tests. if we remove the `if opt.option_strings`, we will face the `ValueError: min() arg is an empty sequence` error. because `opt.option_strings` for subparser is empty list.
Also, if we remove the change `parser._actions` -> `parser_actions` , we will face with `Error: the following arguments are required: --bar` error. because `bar` is required and it is not in the `parser._actions`.